static/src/javascripts/projects/common/modules/commercial/obervable-store.ts

```diff
@@ -0,0 +1,25 @@
+import { onConsentChange } from '@guardian/consent-management-platform';
+import type { ConsentState } from '@guardian/consent-management-platform/dist/types';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { getPageTargeting } from './build-page-targeting';
+
```

Comment:
It seems as though similar to `getInitialConsentState`, it might turn out that the cmp is the best place for `consentObservable` to live.
```suggestion

 * NOTE: depending on where this observable is eventually used, it might be more appropriate
 * for it to be exported from the consent-management-platform
```

Reply:
I decided I didn't need observables in the end but I did introduce [getEnhancedConsent](https://github.com/guardian/frontend/blob/53dd63a9eb897ca813b6f7c452f96ee00582a876/static/src/javascripts/projects/common/modules/commercial/enhanced-consent.ts) which extends the functionality of `getInitialConsentState` so we should align the two at some point.